libs/logic-apps-shared/src/designer-client-services/lib/standard/connection.ts

```diff
@@ -340,7 +341,21 @@ export class StandardConnectionService extends BaseConnectionService implements
 
     const connectionAcls = (await this._getConnectionAcls(connection.id)) || [];
     const { identity, appName } = workflowAppDetails;
-    const identityDetailsForApiHubAuth = this._getIdentityDetailsForApiHubAuth(identity as ManagedIdentity, tenantId as string, identityId);
+    var identityDetailsForApiHubAuth: {principalId: string; tenantId: string};
+    if(isHybridLogicApp(baseUrl)){
+      if (!identity?.principalId || !identity?.tenantId) {
+        throw new Error(
+          intl.formatMessage({
+            defaultMessage: 'App identity is not configured on the logic app environment variables.',
+            id: 'WnU9v0',
+            description: 'Error message when no app identity is added in environment variables',
+          })
+        );
+      }
+      identityDetailsForApiHubAuth = { principalId: identity.principalId, tenantId: identity.tenantId };
+    }else{
```

Comment:
nit: seems to be some formatting issues on this PR

Reply:
Addressed it